### DISCUSSION TITLE
Miten Announcements-kohtaan lähetetään uusi viesti?

### DISCUSSION THREAD

**Discussion by ahomansikka** [voikko/discussion#12] [General]
Minä näen vain mahdollisuuden vastata aiempiin viesteihin.

Uusimmassa vastauksessa, on ilmoitus Sukijan päivityksestä...

**Comment by tlikonen** [comment]
Minulla seuraavan verkkosivun oikeassa reunassa on vihreä painike "New discussion":
https://github.com/orgs/voikko/discussions/categories/announcements

**Reply by ahomansikka** [reply]
Minulle tulee siitä painikkeesta viesti:
Sorry, we didn't recognize that category! Please choose one of the following valid categories.

**Reply by tlikonen** [reply]
Kokeilepa uudelleen. Annoin sinulle lisää oikeuksia.

**Reply by ahomansikka** [reply]
Ei toimi. Tulee edelleen sama ilmoitus.

**Reply by tlikonen** [reply]
Nostin sinua vielä ylöspäin, ylläpitäjäksi (maintainer). Jonkin Githubin oppaan mukaan tuolla tasolla voi käyttää Announcement-kategoriaa.

**Reply by ahomansikka** [reply]
No nyt näkyy Announcement. Kiitämme.

Muuten, kaikki osoitteesta notifications github com tulevat sähköpostit menevät automaagisesti roskakoriin... ei siis roskaposti-kansioon... mutta vika lienee minun sähköpostin asetuksissa. (-:
